packages/gestalt/src/Callout.js

```diff
@@ -179,11 +161,7 @@ export default function Callout({
 
   return (
     <Box
-      dangerouslySetInlineStyle={{
-        __style: {
-          backgroundColor: CALLOUT_TYPE_ATTRIBUTES[type].backgroundColor,
-        },
-      }}
+      color={MESSAGING_TYPE_ATTRIBUTES[type].backgroundColor}
```

Comment:
super nit: access this object once and destructure it

Reply:
DONE